avx2/ntt.h

```diff
@@ -9,19 +9,21 @@ extern const uint32_t zetas_inv[N];
 
 void ntt_levels0t2_avx(uint64_t tmp[N],
                        const uint32_t a[N],
-                       const uint32_t zetas[7]);
+                       const uint32_t zetas[7]) asm("ntt_levels0t2_avx");
```

Comment:
I think it's probably neater to handle this in the assembly files, for example by [a macro](https://github.com/PQClean/PQClean/blob/master/crypto_sign/dilithium3/avx2/cdecl.inc).

Reply:
The pqclean solution seems fine by me.